scripts/globals/monstertpmoves.lua

```diff
@@ -406,12 +406,6 @@ function mobAddBonuses(caster, spell, target, dmg, ele)
     dmg = math.floor(dmg * dayWeatherBonus)
 
     local burst = calculateMobMagicBurst(caster, ele, target)
-
-    -- not sure what to do for this yet
-    -- if (burst > 1.0) then
-        -- spell:setMsg(spell:getMagicBurstMessage()) -- "Magic Burst!"
```

Comment:
probably should have been ability, looks like the intent was handling situations like having shiva burst with blizzard which for avatars isn't a real spell. i agree with the removal though I don't think we want to handle it by trying to get the burst message on a non spell. Or I'm wrong about the intend and its even worse than that. either way :+1: